src/flows/components/panel/InsertCellButton.tsx

```diff
@@ -69,6 +66,65 @@ const InsertCellButton: FC<Props> = ({id}) => {
     )
   }
 
+  if (isFlagEnabled('smallInsert')) {
+    let button
+    if (index === -1) {
+      button = (
+        <div className="insert-wrap" ref={buttonRef}>
+          <span>Insert Panel Here</span>
+          <SquareButton
+            icon={IconFont.Plus_New}
+            size={ComponentSize.ExtraSmall}
+            color={ComponentColor.Secondary}
+            testID={`panel-add-btn-${index}`}
+            className="flow-divider--button"
+            active={popoverVisible.current}
+          />
+        </div>
+      )
+    } else {
+      button = (
+        <div className="insert-wrap" ref={buttonRef}>
+          <span>Insert Panel Below</span>
+          <SquareButton
+            icon={IconFont.ArrowDown_New}
+            size={ComponentSize.ExtraSmall}
+            color={ComponentColor.Secondary}
+            testID={`panel-add-btn-${index}`}
+            className="flow-divider--button"
+            active={popoverVisible.current}
+          />
+        </div>
+      )
+    }
+    return (
+      <div className="small-flow-divider" ref={dividerRef}>
+        {button}
+
+        <Popover
+          enableDefaultStyles={false}
+          appearance={Appearance.Outline}
+          color={ComponentColor.Secondary}
+          triggerRef={buttonRef}
+          position={PopoverPosition.ToTheRight}
+          onShow={handlePopoverShow}
+          onHide={handlePopoverHide}
+          contents={onHide => (
+            <FlexBox
+              direction={FlexDirection.Column}
+              alignItems={AlignItems.Stretch}
+              margin={ComponentSize.Small}
+              className="insert-cell-menu"
+            >
+              <p className="insert-cell-menu--title">Insert Panel Here</p>
+              <AddButtons index={index} onInsert={onHide} />
+            </FlexBox>
+          )}
+        />
```

Comment:
seems like popover is not showing up

Reply:
nvm, after i pull in the updated code, it's working now!

Reply:
did you try with the latest code push? i had the targetref attached to the button like it was before, so if you click on the purple button you get the popup, but when the hover state changes the text color like it does, you expect the text to be clickable too. i tried clicking on the text... no popup... so i changed the targetref to include it

Reply:
lol while i was typing